apis/actions.github.com/v1alpha1/autoscalingrunnerset_types.go

```diff
@@ -250,12 +250,6 @@ type AutoscalingRunnerSetStatus struct {
 	FailedEphemeralRunners int `json:"failedEphemeralRunners"`
 }
 
-func (ars *AutoscalingRunnerSet) ListenerSpecHash() string {
```

Comment:
Note: This does not take into account live patches to the listener itself. If we need to calculate the hash spec live as well, I propose using shorter hash of the values.yaml spec and later add the suffix to the annotation based on the listener spec hash